pkg/controllers/node/node_syncer.go

```diff
@@ -63,14 +63,14 @@ func (c *NodeController) OnUpdates(updates []bapi.Update) {
 		case bapi.UpdateTypeKVUpdated:
 			n := upd.KVPair.Value.(*apiv3.Node)
 
-			if c.config.SyncLabels {
```

Comment:
This change just makes sure we're always populating the nodemapper map (we're already always running the syncer.)